PYR1-1157.org

```diff
@@ -0,0 +1,8 @@
+* TODO only show WS-icons&info to Authenticated users.
```

Comment:
Don't worry about doing this here. I'm going to do it in https://sig-gis.atlassian.net/browse/PYR1-1181 given that the user roles have changed.